molecule/ubuntu2204/molecule.yml

```diff
@@ -33,17 +36,15 @@ provisioner:
     converge: ../resources/playbook.yml
   inventory:
     host_vars:
-      omero-server-ubuntu1804:
+      omero-server-ubuntu2204:
         postgresql_version: "11"
```

Comment:
seems old.

Reply:
Sorry, I closed this because there were conflicts. New PR is https://github.com/ome/ansible-role-omero-server/pull/69.